int32 code

int32 subsystem

int32 urgency

string message

time stamp
